### DISCUSSION TITLE
Request: Add boolean output to Text Find and Replace

### DISCUSSION THREAD

**Discussion by albamuth** [WASasquatch/was-node-suite-comfyui#214] [Ideas]
It would be helpful to return true or false if the find/replace occured. Also might be helpful to return an integer for the amount of find/replacements that occcured - or just return an int since no replacements would result in 0?

Application: I'm use a WD14 tagger for a batch img-to-img conversion, and would like to change the sampler denoise value if the string "close_up" is ever detected. I'm having trouble finding a simple string-inclusion node anywhere and I think simply adding on a boolean/number output to your find/replace node would be pretty easy.

**Comment by WASasquatch** [comment]
Does sound helpful. Yeah I can add that.

**Reply by albamuth** [reply]
You're the best! Thanks in advance.

**Comment by WASasquatch** [comment]
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/b15b2a67-2b36-4eae-8fae-bc76b3a15829)
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/211de466-222e-4d81-a501-24e211ae3f53)

Added here: https://github.com/WASasquatch/was-node-suite-comfyui/commit/b59970f09f4d7afdb180136870032ef3fe8e68aa

**Comment by albamuth** [comment]
Nice! This is will be very useful.
